go.mod

```diff
@@ -1,13 +1,14 @@
 module github.com/qdrant/migration
 
-go 1.24
+go 1.24.1
```

Comment:
is this change required within the scope of this PR? IMO it could be done separately.